lib/closePR.js

```diff
@@ -16,7 +16,7 @@ module.exports = async ({ dependents }) => {
       const branch = await context.getTestingBranchName(parentBranchName)
       const resp = await github.getChecks(dependentPkgInfo.owner, dependentPkgInfo.name, branch)
       const closedPR = await closeDependencyPR(dependentPkgInfo.owner, dependentPkgInfo.name, branch, resp.data.check_runs)
-      if (closedPR) {
+      if (closedPR && closedPR.length > 0) {
```

Comment:
in that case, is it ever *not* an array?
```suggestion
      if (closedPR.length > 0) {
```

Reply:
yes, that is true

Reply:
Seems the tests are now failing?

I haven't checked if this is a mocked response...